server/services/licenceService.ts

```diff
@@ -580,4 +580,17 @@ export default class LicenceService {
   async getPrisonerDetail(nomsId: string, user: User): Promise<CaseloadItem> {
     return this.licenceApiClient.getPrisonerDetail(nomsId, user)
   }
+
+  async searchPrisonersByNomsIds(nomsIds: string[], user: User): Promise<CaseloadItem[]> {
+    return this.licenceApiClient.searchPrisonersByNomsIds(nomsIds, user)
+  }
+
+  async searchPrisonersByReleaseDate(
+    earliestReleaseDate: Date,
+    latestReleaseDate: Date,
+    prisonIds?: string[],
+    user?: User
+  ): Promise<CaseloadItem[]> {
+    return this.licenceApiClient.searchPrisonersByReleaseDate(earliestReleaseDate, latestReleaseDate, prisonIds, user)
+  }
```

Comment:
need to remove equivalents from prisoner search api
need to look into removing bankholiday util functions

Reply:
and update specs to use new payload

Reply:
need to test these

Reply:
Will remove the rest of the stuff in a follow up